### DISCUSSION TITLE
Convenience "run current query" button in-editor?

### DISCUSSION THREAD

**Discussion by 4JX** [mtxr/vscode-sqltools#1181] [Q&A]
Does something like the "run on active connection" button exist but only for the current highlighted/selected query? I'm aware the command palette method exists, but I feel it would be more convenient to have a direct button to press

**Comment by gjsjohnmurray** [comment]
Are you not seeing CodeLens links appear above your query?

![image](https://github.com/mtxr/vscode-sqltools/assets/6726799/7e3eb98d-308a-492d-8ccb-6276342c375c)

See also https://vscode-sqltools.mteixeira.dev/en/features/codelens

**Comment by gjsjohnmurray** [comment]
No response from OP, so closing this.
